apps/alchemist/lib/alchemist/topic_manager.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Alchemist.TopicManager do
+  @moduledoc """
+  Create Topics in kafka using the Elsa library.
+  """
+  use Retry
+  use Properties, otp_app: :alchemist
+
+  getter(:retry_initial_delay, generic: true)
+  getter(:retry_count, generic: true)
+  getter(:elsa_brokers, generic: true)
+  getter(:output_topic_prefix, generic: true)
+  getter(:input_topic_prefix, generic: true)
+
+  @spec setup_topics(%SmartCity.Ingestion{}) :: %{input_topic: String.t(), output_topic: String.t()}
+  def setup_topics(ingestion) do
+    input_topic = input_topic(ingestion.id)
+    output_topic = output_topic(ingestion.id)
```

Comment:
Alchemist outputs onto its dataset's topic, not an ingestion.

Reply:
Corrected https://github.com/UrbanOS-Public/smartcitiesdata/pull/1329/commits/adba5ec3ebcbc87f54a90cd4cc2ecbddc62a10ea